index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <header class="">
+      <section class="">
+        <h1>Hi, I'm Ashwini!</h1>
+        <h4>Apprentice at Microsoft</h4>
+      </section>
+      <nav class="nav-main">
+      <div class="icon-menu"></div>
+      <ul>
+        <li class="tab" data-content="articles"><a href="#" class="icon-home"> Home</a></li>
+        <li class="tab" data-content="about"><a href="https://ashkrao.github.io/about_me/" class="icon-address-book"> About</a></li>
```

Comment:
We want this project to be a single page application, separate from previous work. I would suggest recreating an about me section on your portfolio!